### PR TITLE
Update the user-timing IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

File: interfaces/user-timing.idl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+        partial interface Performance {
```
Comment (foolip) [inline interfaces/user-timing.idl]: Can you update to strip the indentation and use the file in https://github.com/w3c/web-platform-tests/blob/master/user-timing/idlharness.html?

**Comment by foolip** [comment]
Hmm, and why is Travis unhappy?

File: interfaces/user-timing.idl (hunk 1 above)
Comment (foolip) [reply interfaces/user-timing.idl]: @lukebjerring, indentation problem still persists.

**Comment by foolip** [comment]
@lukebjerring looks like this needs a test to exercise the IDL?

**Comment by lukebjerring** [comment]
I've updated the `user-timing/idlharness.html` test to be `.any.js`, since it's exposed on both worker + window, and changed the test to use the IDL. PTAL.

**Comment by lukebjerring** [comment]
cc @npm1 who was helping me work out whether using `.any.js` was the right thing to do. Note that we see a large number of `.any.worker.html` results with:

> assert_own_property: self does not have own property "PerformanceMark" expected property "PerformanceMark" missing

Goes a bit beyond my understanding of the implementation of idlharness. @foolip - any idea whether that's to be expected?

File: user-timing/idlharness.any.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// META: global=window,worker
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+promise_test(async () => {
+  const idl_array = new IdlArray();
+  const idl = await fetch("/interfaces/user-timing.idl").then(r => r.text());
+  const hrtime = await fetch("/interfaces/hr-time.idl").then(r => r.text());
```
Comment (plehegar) [inline user-timing/idlharness.any.js]: you need to import /interfaces/performance-timeline.idl as well, since PerformanceMark depends on PerformanceEntry.

Comment (lukebjerring) [reply user-timing/idlharness.any.js]: Alright, I'll pull in https://github.com/web-platform-tests/wpt/pull/9809 to this PR.

**Review by foolip — APPROVED** [review]
Tested locally, .worker.html tests will reveal some things that don't match the spec in Chromium :) :+1:

**Pull request merged**